pjmedia/src/pjmedia/transport_srtp_dtls.c

```diff
@@ -1676,12 +1674,10 @@ static pj_status_t dtls_encode_sdp( pjmedia_transport *tp,
         {
             char addr[PJ_INET6_ADDRSTRLEN];
             char addr2[PJ_INET6_ADDRSTRLEN];
-            PJ_LOG(2,(ds->base.name, "Attached transport, remote addr=%s:%d "
-                                     "remote rtcp=%s:%d",
-                      pj_sockaddr_print(&ap.rem_addr, addr, sizeof(addr), 2),
-                      pj_sockaddr_get_port(&ap.rem_addr),
-                      pj_sockaddr_print(&ap.rem_rtcp, addr2, sizeof(addr2), 2),
-                      pj_sockaddr_get_port(&ap.rem_rtcp)));
+            PJ_LOG(2,(ds->base.name, "Attached transport, remote addr=%s "
+                                     "remote rtcp=%s",
+                      pj_sockaddr_print(&ap.rem_addr, addr2, sizeof(addr), 3),
```

Comment:
Should be:
```
pj_sockaddr_print(&ap.rem_addr, addr, sizeof(addr), 3),
pj_sockaddr_print(&ap.rem_rtcp, addr2, sizeof(addr2), 3)));
```

Reply:
the usage of addr and addr2 is intentionally reversed here to avoid exceeding 80 columns.